vsphere/datadog_checks/vsphere/data/conf.yaml.example

```diff
@@ -110,20 +110,29 @@ instances:
   ## 1: Only basic metrics - 4: every metric available.
   ## Warning: Depending on the size of the vSphere environment, metric collection can be slow,
   ## very CPU intensive and put pressure on the vCenter Server
-  # 
+  #
   #  collection_level: 1
 
-  ## @param event_config - dictionary - optional 
+  ## @param use_guest_hostname - boolean - optional - default: false
+  ## If true, the check will use the guest hostname for VMs instead of the VM name
+  ## This requires the VM to have VM tools installed in it. If the guest hostname is
```

Comment:
vm tools?

Reply:
https://docs.vmware.com/en/VMware-vSphere/6.7/com.vmware.vsphere.html.hostclient.doc/GUID-28C39A00-743B-4222-B697-6632E94A8E72.html

Reply:
Should we call them `VMware Tools` like in the docs?